src/Signin.tsx

```diff
@@ -112,20 +112,26 @@ export class Signin extends PureComponent<Props, State> {
       } = this.props
 
       clearInterval(this.intervalID)
-      /**
-       * We'll need this authSessionCookieOn flag off for tools until
-       * Quartz is integrated into that environment
-       */
-      if (CLOUD && isFlagEnabled('authSessionCookieOn')) {
-        const url = new URL(
-          `${window.location.origin}${CLOUD_LOGIN_PATHNAME}?redirectTo=${window.location.href}`
-        )
-        setToLocalStorage('redirectTo', window.location.href)
-        window.location.href = url.href
-        throw error
-      }
 
       if (CLOUD) {
+        if (isFlagEnabled('useQuartzLogin')) {
+          window.location.reload()
+          throw error
```

Comment:
like we discussed offline, I think we should remove this line since it won't really do anything and it tends to communicate that is doing something